llvm/include/llvm/IR/IntrinsicInst.h

```diff
@@ -341,6 +341,14 @@ class DbgVariableIntrinsic : public DbgInfoIntrinsic {
     return getIntrinsicID() == Intrinsic::dbg_declare;
   }
 
+  /// Does this describe the value of a local variable. True for dbg.value,
+  /// but False for dbg.declare, which describes its address, and False for
```

Comment:
```suggestion
  /// but false for dbg.declare, which describes its address, and false for
```
this is not Python :-)